extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/handler/ZonePlayerHandler.java

```diff
@@ -2801,4 +2815,8 @@ private long sleepStrTimeToSeconds(String sleepTime) {
         int seconds = Integer.parseInt(units[2]);
         return 3600 * hours + 60 * minutes + seconds;
     }
+    
+    protected Integer getNotificationTimeout() {
+        return this.notificationTimeout != null ? this.notificationTimeout : DEFAULT_NOTIFICATION_TIMEOUT;
```

Comment:
This safety check is not necessary (and the getter a standard getter method) if you initialize the default in the initialize() method (see my previous comment)